content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
 icon beside the first activity in the workflow. When clicked, the page reloads, and Odoo shows the
 various results (and analytics) connected to that specific activity as they occur, in real-time.
 
+.. note::
+   If a child activity is schedule beneath a parent activity, that child activity will be revealed
+   in the workflow, once that parent activity has been run, via the :guilabel:`▶️ (play button)`
+   icon.
+
 .. image:: testing_running/workflow-test-progress.png
    :align: center
    :alt: Workflow test progress in Odoo Marketing Automation.
 
-Once all the workflow activities are completed, the test will end, and move to the
-:guilabel:`Completed` stage. To stop a test before all the workflow activities are completed, click
-the :guilabel:`Stop` button in the upper-left corner of the campaign test page.
+Once all the workflow activities are completed, the test ends, and the status bar (in the
+upper-right corner) moves to the :guilabel:`Completed` stage.
+
+To stop a test before all the workflow activities are completed, click the :guilabel:`Stop` button
+in the upper-left corner of the campaign test page.
+
+Running campaigns
+=================
+
+To run a campaign, navigate to :menuselection:`Marketing Automation app`, and select the desired
+campaign to run.
+
+On the campaign detail form, with all the desired activities ready in the :guilabel:`Workflow`
+section, click :guilabel:`Start` in the upper-right corner to officially run the campaign to the
+configured target audience specified on the campaign detail form.
+
+.. image:: testing_running/start-button-campaign-detail-form.png
+   :align: center
+   :alt: The Start button in the upper-left corner of the marketing campaign detail form.
+
+Clicking :guilabel:`Start` launches the campaign, and the status bar of the campaign switches to
+:guilabel:`Running`, which is located in the upper-right corner of the campaign detail form.
+
+.. image:: testing_running/campaign-running-status.png
+   :align: center
+   :alt: The status of a marketing campaign changing to running in the upper-right corner.
+
+.. note::
+   If some participants are already running on a campaign, and was stopped for any reason, when the
+   :guilabel:`Start` button is clicked again, a pop-up warning appears. The pop-up warning instructs
+   the user to click an :guilabel:`Update` button to apply modifications to the campaign that have
+   been made.
+
+   .. image:: testing_running/workflow-modification-warning.png
+      :align: center
+      :alt: The workflow has been modified warning pop-up window of a marketing campaign form.
+
+   Be aware that participants that had no more activities left to recieve/interact with in the
+   campaign, **can** be reintroduced into the campaign, and new traces could be created for them.
```

Comment:
What does this mean in layman's terms? This is a repeat of what the screenshot says.